pkgs/by-name/_7/_7-segment-font/package.nix

```
@@ -0,0 +1,34 @@
+{
+  stdenvNoCC,
+  fetchurl,
+  lib,
+}:
+stdenvNoCC.mkDerivation {
+  pname = "7-segment-font";
+  version = "0-unstable-2025-02-09";
+
+  src = fetchurl {
+    url = "https://web.archive.org/web/20250209170057/https://torinak.com/font/7segment.ttf";
+    hash = "sha256-8OnWHYNahx7meaJ1LuZNmQKGj/606x9yF/CDdfOrjjE=";
+  };
+
+  phases = [ "installPhase" ];
```

This is deprecated, and shouldn't have to set it manually
```suggestion
  dontUnpack = true;
  dontBuild = true;
```